### DISCUSSION TITLE
Adult listings

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#94] [Show and tell]
I do **not** know at all what this service is

https://telegra.ph/listing---rstklmxyz--adult-channels-only-05-16

**Comment by freearhey** [comment]
Links look similar to these: https://telegra.ph/vpnpool-11-19

**Reply by BellezaEmporium** [reply]
Indeed, there's lots of alternatives to those. Though I do not seem to know from where they come from.

EDIT : I confirm that listing is the same.
